Dynatrace/dynatrace-operator
# Prunes the makefile

# Description
Renames/refactors/prunes things in the makefile.

Removed commands that didn't work (properly) (or didn't make sense)
Introduced new targets instead of using `if`s and envvars
Renamed targets that were too vague, inaccurate or unhelpful

Didn't touch the image setting/switching logic. (I couldn't come up with a better solution)

## How can this be tested?
Test out the commands, see if they work.
If removed one of your favourite command, pls comment which one was it and why do you use it 😄   


## Checklist
